helm/charts/determined/templates/master-permissions.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
      release: {{ .Release.Name }}
 rules:
   - apiGroups: [""]
-    resources: ["pods", "pods/status", "pods/log", "configmaps", "namespaces"]
+    resources: ["pods", "pods/status", "pods/log", "configmaps"]
```

Comment:
I'm just curious - does this give master permission to create the resources in the list?

Reply:
Yea! The pods in the master deployment running the master service are authorized as the specified service account when accessing the Kubernetes API and have the permissions granted to the service account in the helm release, among those permissions being the ability to CRUD the listed resources!

Reply:
Oh, awesome - thanks!!